.gitignore

```diff
@@ -1,5 +1,4 @@
 # Special build files
-engine/overrideversion.go
```

Comment:
Hm, I wasn't actually aware this was in there. I assume that some devs used this to manually set the version instead of using ldflags. Any specific reason for removing it from .gitignore?

Reply:
Just because I removed the ` scripts/prebuild.sh ` that was generating this file, so a priori that file should never show up again. That's seems to me just a hacky way of doing what ldflag can do,

Reply:
Ah, some GoLand thing that I'm not using. It's okay with me then but I have no idea if anybody depends on it for whatever reason.

Reply:
I have restored this prebuild script and re added to the git ignore